src/common/components/nav/desktopNav.tpl.html

```diff
@@ -0,0 +1,124 @@
+<header id="desktop-navigation" class="gh-desk-nav">
+  <div class="container">
+
+    <a href="" title="Cru" class="gh-desk-logo"></a>
+
+    <ul class="gh-desk-nav-tree col-level-1">
+      <li ng-repeat="i in $ctrl.menuStructure.main" ng-class="{'has-children': i.children.length, 'open-menu': $ctrl.mobileMenuPath.main[0] === i.path.split('/').pop()}"
```

Comment:
I think it is really hard to know what `i` is here. You've used it for 3 different things. The code works fine, it is just hard to read as you have to go find the nearest ng-repeat to see what it is but then `ng-repeat="i in i.children"` really doesn't tell you how deep you are. Maybe it could be `menuItemLevel1`, `itemLevel1`, `itemLvl1`, `item1` or something similar. I think sometimes I end up making things too verbose but I don't really like single character variable names. Up to you.